website/blog/2024/11-10-web-scraping-tips/index.md

```diff
@@ -251,8 +255,9 @@ data = {
 }
 
 response = requests.post(url, json=data)
-
+# highlight-start
```

Comment:
I think this is more of a debugging relict (it would make more sense to call `.json()` here too - the response is still JSON document). Feel free to swap it to `.json()` (and remove the highlighting).